callstack/react-native-paper
# docs: add missing screenshots

### Motivation


 #1834
I added several missing components screenshots:
- [x] Divider
- [x] RadioButton.Item
- [x] Badge
- [x] List.AccordionGroup

I've also spotted two docs rendering bugs that were easy to fix. Now the Checkbox.Item and List.AccordionGroup display correctly.

### Test plan

1. Open docs, verify the components mentioned above have screenshots.
2. Verify that List.AccordionGroup now displays its usage example.
3. Verify that RadioButton is not duplicated and that Checkbox includes Checkbox.Item.
